src/graphdb.js

```diff
@@ -150,36 +184,60 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
         r += `input ${nodeCase}Input {\n`;
         r += '\t_id: ID @id\n';
         node.properties.forEach(property => {
-            r+= `\t${property.name}: ${property.type}\n`;
+            let propertyCase = replaceCleanseLabel(property.name);
+            if (property.name !== propertyCase) {
+                r+= `\t${propertyCase}: ${property.type} @alias(property: "${property.name}")\n`;
+            }
+            else {
+                r+= `\t${property.name}: ${property.type}\n`;
+            }
         });
         r += '}\n\n';
     })
 
     // edge types
     gdbs.edgeStructures.forEach(edge => {
         // edge type
-        let edgeCase = edge.label;
+        let edgeCase = replaceCleanseLabel(edge.label);
         if (changeCase) {
-            edgeCase = toPascalCase(edge.label);
-            r += `type ${edgeCase} @alias(property:"${edge.label}") {\n`;        
-        } else {
+            edgeCase = toPascalCase(edgeCase);
+        }
+        if (edge.label !== edgeCase) {
+            r += `type ${edgeCase} @alias(property:"${edge.label}") {\n`;  
+        }
+        else {
             r += `type ${edgeCase} {\n`;
         }
         r += '\t_id: ID! @id\n';
 
         edge.properties.forEach(property => {
-            if (property.name == 'id')                
-                r+= `\tid: ID\n`;
-            else
-                r+= `\t${property.name}: ${property.type}\n`;
+            if (property.name == 'id') {
+                r += `\tid: ID\n`;
+            }
+            else {
+                let propertyCase = replaceCleanseLabel(property.name);
+                let alias = '';
+                if (property.name !== propertyCase) {
+                    alias = ` @alias(property: "${property.name}")`;
+                }
+                r+= `\t${propertyCase}: ${property.type}${alias}\n`;
+            }
         });
         r += '}\n\n';
 
         // input for the edge type
-        if (edge.properties.length > 0) {            
-            r += `input ${edgeCase}Input {\n`;        
+        if (edge.properties.length > 0) {       
+            let edgeCase = replaceCleanseLabel(edge.label);
```

Comment:
This is already done at line 201-204 so you can just reference the existing `edgeCase` variable.
```
let edgeCase = replaceCleanseLabel(edge.label);
        if (changeCase) {
            edgeCase = toPascalCase(edgeCase);
        }
```